src/c++/perf_analyzer/data_loader.cc

```diff
@@ -481,22 +479,26 @@ DataLoader::ReadTensorData(
           }
         }
 
-        if (!tensor.HasMember("content")) {
-          return cb::Error(
-              "missing content field. ( Location stream id: " +
-                  std::to_string(stream_index) +
-                  ", step id: " + std::to_string(step_index) + ")",
-              pa::GENERIC_ERROR);
-        }
+        if (tensor.HasMember("b64")) {
```

Comment:
The diff looks weird, but this fixes a bug for the 'b64' case we were not capturing the user-supplied shape.